core/src/main/java/org/jboss/set/mjolnir/archive/mail/MailingService.java

```diff
@@ -0,0 +1,73 @@
+package org.jboss.set.mjolnir.archive.mail;
+
+import javax.annotation.Resource;
+import javax.ejb.Stateless;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.*;
+
+@Stateless
+public class MailingService {
+
+    private String to;
+
+    private String from;
+
+    private String subject;
+
+    private String body;
+
+    //TODO: Add SMTP server
+    @Resource (name="")
```

Comment:
I'm not sure how to produce this resource for tests, let me know if you need help. Theoretically just creating CDI producer method annotated with `@Produces @Resource(lookup="java:...")` could be enough, according to https://docs.jboss.org/weld/reference/2.0.4.Final/en-US/html_single/#resources.

The method would probably create a mock (like in `UnitTestProducer#createArchivingBeanMock()`), and in tests we could check that the send() method was called with correct arguments (like in https://github.com/jboss-set/mjolnir-archive-service/blob/master/core/src/test/java/org/jboss/set/mjolnir/archive/batch/MembershipRemovalBatchletTest.java#L137).

Reply:
Or maybe we don't need to implement the producer, just set the instance directly from the test. I'm not sure if the existing CDI enabled test case will fail because of missing bean.